pkg/feature/manifest.go

```diff
@@ -44,6 +50,22 @@ func (b *rawManifest) Process(_ any) ([]*unstructured.Unstructured, error) {
 	return convertToUnstructuredSlice(resources)
 }
 
+func (b *rawManifest) SetManaged(objs []*unstructured.Unstructured) []*unstructured.Unstructured {
+	if b.patch {
+		return objs
+	}
+	for _, obj := range objs {
+		objAnnos := obj.GetAnnotations()
+		if objAnnos == nil {
+			objAnnos = make(map[string]string)
+		}
+
+		objAnnos[annotations.ManagedByODHOperator] = annotationTrue
+		obj.SetAnnotations(objAnnos)
+	}
+	return objs
+}
```

Comment:
```suggestion
func (b *rawManifest) SetManaged(objects []*unstructured.Unstructured) []*unstructured.Unstructured {
	if !b.patch {
		markAsManaged(objects)
	}
	return objects
}
```

Reply:
I was thinking that maybe instead of extracting `annotationTrue` we could actually reuse the whole thing twice :) (see below)